dataland-frontend/package.json

```diff
@@ -59,6 +59,7 @@
     "mocha": "^10.2.0",
     "npm-check-updates": "^16.14.4",
     "otplib": "^12.0.1",
+    "sass": "^1.67.0",
```

Comment:
Why was this added?

Reply:
@fsoffel you mean the node version? only @types/node was updated not node itself, I updated because `updatepackages` suggested it and nothing broke,  I can downgrade to previous if needed.

Reply:
@fsoffel sass was required dependency for vite at least for the latest version

Reply:
Dont packages manage that by themselves in the package-lock.json?